deming_matthew_resources.md

```diff
@@ -0,0 +1,8 @@
+Favorite Resources
```

Comment:
What do you think about making this an H1 (`# Favorite Resources`)?

I like the use of bullets. `* > -` imo.

(Just some markdown nitpickin' and demonstrating how PR reviews look. No changes needed)